modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Runtime/StandaloneCliOptions.cs

```diff
@@ -330,6 +330,11 @@ public class StandaloneCliOptions : Dictionary<string, string>, IAgentConfigProv
         /// </summary>
         public int? MaxOutgressMessages => StandaloneCliModel.MaxOutgressMessages;
 
+        /// <summary>
+        /// Flag to use reversible encoding for messages
+        /// </summary>
+        public bool? UseReversibleEncoding => StandaloneCliModel.UseReversibleEncoding;
```

Comment:
`UseReversibleEncoding` has no configuration surface. We should add command line parameter for it and add propagation of the value in `ToStandaloneCliModel()` method.

Reply:
Added in #1694.